sound/soc/sof/sof-audio.h

```diff
@@ -458,6 +458,9 @@ struct snd_sof_widget {
 	struct ida output_queue_ida;
 	struct ida input_queue_ida;
 
+	/* optionally provided by the topology pipeline KCPS value */
+	unsigned int pipeline_kcps;
```

Comment:
this is a very ipc4 specific parameter, can we put it somewhere in one of the ipc4 structs instead of swidget?

Reply:
moved, thanks